kf_lib_data_ingest/common/constants.py

```diff
@@ -78,53 +106,106 @@ class REFERENCE_GENOME:
         HG19 = 'hg19'
 
     class PLATFORM:
+        GENOMICS = 'Complete Genomics'
         ILLUMINA = 'Illumina'
+        ION_TORRENT = 'Ion Torrent'
+        LS454 = 'LS454'
+        PACBIO = 'PacBio'
+        SOLID = 'SOLiD'
 
     class INSTRUMENT:
         HISEQ_X_v2_5 = 'HiSeq X v2.5'
         HISEQ_X_10 = 'HiSeq X Ten'
 
     class STRAND:
-        UNSTRANDED = 'Unstranded'
         FIRST = 'First Stranded'
         SECOND = 'Second Stranded'
-        OTHER = 'Other'
+        UNSTRANDED = 'Unstranded'
 
     class CENTER:
-        class WASHU:
-            NAME = 'Washington University'
-            KF_ID = 'SC_K52V7463'
+        class BAYLOR:
+            NAME = 'Baylor College of Medicine'
+            KF_ID = 'SC_A1JNZAZH'
+
+        class BC_CANCER_ANGECY:
+            NAME = 'British Columbia Cancer Agency Genome Sciences Centre'
+            KF_ID = 'SC_FN7NH453'
+
+        class BGI:
+            NAME = 'BGI@CHOP Genome Center'
+            KF_ID = 'SC_WWEQ9HFY'
 
         class BROAD:
             NAME = 'Broad Institute'
             KF_ID = 'SC_DGDDQVYR'
 
+        class FELINE_DIAGNOSTICS:
+            NAME = 'Feline Diagnostics LLC'
+            KF_ID = 'SC_CATTVETT'
+
         class HUDSON_ALPHA:
             NAME = 'HudsonAlpha Institute for Biotechnology'
             KF_ID = 'SC_X1N69WJM'
 
-        class BAYLOR:
-            NAME = 'Baylor College of Medicine'
-            KF_ID = 'SC_A1JNZAZH'
+        class NANT:
+            NAME = 'NantOmics'
+            KF_ID = 'SC_N1EVHSME'
+
+        class NOVOGENE:
+            NAME = 'Novogene'
+            KF_ID = 'SC_2ZBAMKK0'
+
+        class NCI:
+            NAME = 'National Cancer Institute, Khan Lab'
+            KF_ID = 'SC_F6RZ51K9'
+
+        class SIDRA:
+            NAME = 'Genomic Clinical Core at Sidra Medical and Research Center'
+            KF_ID = 'SC_KE2ASNJM'
 
         class ST_JUDE:
             NAME = 'St Jude'
             KF_ID = 'SC_1K3QGW4V'
 
+        class TGEN:
+            NAME = 'The Translational Genomics Research Institute'
+            KF_ID = 'SC_KQ9JZG3P'
+
+        class VIRTUAL:
+            NAME = 'Virtual'
```

Comment:
What is this?

Reply:
For CBTTC, there are new data types:
```
HISTOLOGY_IMAGES = 'Histology Images'
OPERATION_REPORTS = 'Operation Reports'
PATHOLOGY_REPORTS = 'Pathology Reports'
RADIOLOGY_IMAGES = 'Radiology Images'
RADIOLOGY_REPORTS = 'Radiology Reports'
```

Since these are not actual genomic files, I had to create placeholding BSs with `analyte_type` `Virtual` which is required. In turn, the `VIRTUAL` SC is for these virtual samples since SC is required for BS.